autoload/slime.vim

```diff
@@ -25,7 +25,8 @@ function! s:ScreenSend(config, text)
         \ " -X eval \"readreg p " . g:slime_paste_file . "\"")
   call system("screen -S " . shellescape(a:config["sessionname"]) . " -p " . shellescape(a:config["windowname"]) .
         \ " -X paste p")
-  call system('screen -X colon "
"')
+  call system('screen -X colon "
+"')
```

Comment:
Something seems to have happened here, think I had the same thing with my editor wanting to change this.

Reply:
Thanks for catching it. Don't know what happened there.

Reply:
Reverted in the next commit